libqtile/backend/x11/core.py

```diff
@@ -427,7 +427,7 @@ def grab_key(self, key: Union[config.Key, config.KeyChord]) -> Tuple[int, int]:
 
         for code in codes:
             if code == 0:
-                logger.warning(
+                logger.debug(
```

Comment:
This still feels like a warning, the user asked for something and they didn't get it, right?

Reply:
> If it can't make sense to your grandma, it's probably meant to be a DEBUG message.

Reply:
> I don't know, that's kind of a nonsense heuristic. My grandmother has late stage dementia, most of the world doesn't make sense to her.

And I don't have a grandma anymore. That was a metaphor. :)

> How will people find out their keys weren't correctly mapped without this message? I'm imagining they'll file a bug, we'll tell them to run it with --debug, and then they'll see this message. Seems like we could short circuit that process by leaving it as a warning.

Nobody knows what a keysym and a modmask are, especially when printed as hex, so I don't think it would short circuit anything. That being said I don't mind much because I'm not even sure it's possible to enter that branch ; invalid keys will return a ConfigError and fallback to the default config long before we end up here. So I went back to a warning message, but tried to make it a bit more comprehensive. Does it look good to you?

Reply:
I'm not 100% sure it's dead though. :)

Reply:
Actually I do trigger it when I bind `` ` `` but... strangely enough, the key still works. :D

```
Can't grab <Key (['mod4'], dead_grave)> (unknown keysym: 0xfe50)
Can't grab <Key (['mod4', 'shift'], dead_grave)> (unknown keysym: 0xfe50)
```

Reply:
Maybe there are 2 codes that that key, and one is working, one is hitting the log?